app/objects/secondclass/c_link.py

```diff
@@ -60,6 +60,8 @@ class Link(BaseObject):
     load_schema = LinkSchema(exclude=['decide', 'pid', 'facts', 'unique', 'collect', 'finish', 'visibility',
                                       'output'])
 
+    RESERVED = dict(origin_link_id='#{origin_link_id}')
```

Comment:
would it make more sense to just call this value `#{link_id}` ?  If I'm understanding correctly, the value that gets instantiated into a link's command will be the link's id.

Reply:
Your understanding is correct, I thought it would make sense to keep the value name the same here as it's named in the sandcat command line argument. I have no problem changing this to just `#{link_id}` instead and keeping the sandcat argument the way it is, unless you think it makes sense to change the name to `link_id` across the board?

Reply:
hmm I think `origin_link_id` is great name on the Agent.  Just wasn't sure if it was as intuitive on links.  The way it is works for me, but feel free to push up the change to use `#{link_id}` in agent command-lines if you think it's clearer.